### ISSUE TITLE
Empty options

### ISSUE THREAD

**Issue by Zonzei** [AdiAddons/AdiBags#196]
For while now every time I try to open Adibags options I only get empty options panel and error.

Doesn't seem to matter what addons I have enabled since same thing happens with only Adibags enabled or whatever mix of my regular addons enabled/disabled. [Here](http://imgur.com/a/nSdGA) is a screenshot of the empty options panel and the error itself with only Adibags and BugGrabber enabled. Using latest version of Adibags of course. 

Using search revealed that this has been issue for others in the past but I couldn't find solutions.

`39x ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:376: invalid order function for sorting
[C]: ?
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:376: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:344>
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:969: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:965>
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:988: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:965>
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:1021: in function <...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:999>
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:1675: in function `FeedGroup'
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0-61.lua:1868: in function `Open'
AdiBags\config\Options-Options.lua:685: in function `OpenOptions'
AdiBags\core\Core.lua:295: in function <AdiBags\core\Core.lua:289>
(tail call): ?
AdiBags\modules\FilterOverride.lua:496: in function `?'
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-6.lua:145: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:145>
[string "safecall Dispatcher[4]"]:4: in function <[string "safecall Dispatcher[4]"]:4>
[C]: ?
[string "safecall Dispatcher[4]"]:13: in function `?'
...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0-6.lua:90: in function <...Ons\Ace3\CallbackHandler-1.0\CallbackHandler-1.0.lua:85>
(tail call): ?

Locals:
nil`

**Comment by Talyrius** [comment]
Are you sure those are the only addons enabled? It looks like you may have an older version of Ace3 that has been modified to supersede newer revisions.

**Comment by Zonzei** [comment]
Pretty sure since I used the "Disable all" button on login screen addon selection, scrolled through the list to make sure nothing was selected and after that only selected Adibags + Adibags config.

**Comment by Talyrius** [comment]
Try redownloading v1.9.6 from [here](https://www.wowace.com/projects/adibags/files/2401502) (not the "-nolib" version).

**Comment by Zonzei** [comment]
Still same thing happening with that one.

**Comment by Pandyana** [comment]
I'm having the same issue. I can't open the configuration menu. If I try to open it I get the following bug report and a blank window. I have tried uninstalling the addon and then reinstalling it. Turning some off and turning some on. I'm not sure what I did or when I did it. I noticed the currency on the bottom of the bag addon was weird so I went to see what was up in the config options and this is what I now get. I can't tell you when it started happening because I haven't opened the options in a really long time.
![wowscrnshot_043017_204121](https://cloud.githubusercontent.com/assets/28242616/25570089/1e36718c-2de7-11e7-849f-1ce9b007b769.jpg)

Here's the error I'm getting:
Date: 2017-04-30 20:32:27
ID: 1
Error occured in: Global
Count: 1
Message: ...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua line 376:
invalid order function for sorting
Debug:
[C]: ?
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:376:
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:344
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:969:
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:965
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:988:
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:965
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1021:
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:999
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1675: FeedGroup()
...nfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:1868: Open()
AdiBags\config\Options.lua:685: OpenOptions()
AdiBags\core\Core.lua:295:
AdiBags\core\Core.lua:289
(tail call): ?
(tail call): ?
Locals:
(*temporary) =

{
1 = "Ancient Mana"
2 = "Artifact Power"
3 = "Fish Bait"
4 = "Recipe"
5 = "Reagent"
6 = "Broken Shore"
7 = "Champion Upgrades"
8 = "Junk"
9 = "Garrison"
10 = "Reputation"
11 = "Miscellaneous"
12 = "Equipment"
13 = "Consumable"
14 = "newAssoc"
15 = "Toy"
16 = "description"
17 = "Quest"
18 = "Trade Goods"
}
(*temporary) = defined @Interface\AddOns\Masque\Libs\AceConfig-3.0\AceConfigDialog-3.0\AceConfigDialog-3.0.lua:314
(*temporary) = "Junk"
(*temporary) = "Junk"
(*temporary) = "Artifact Power"
AddOns:
KalielsTracker, v2.1.5
MoncaiCompare, v7.0.1
Swatter, v7.4.5714 (TasmanianThylacine)
AchieveIt, v
AdiBags, vv1.9.6
AdiBagsConfig, vv1.9.6
AdiBagsLegionFilters, v1.5.4
AlreadyKnown, v1.13
AraBrokerGuildFriends, vr75
AraBrokerReputations, vr18
AucAdvanced, v7.4.5714 (TasmanianThylacine)
AucFilterBasic, v7.4.5714 (TasmanianThylacine)
AucFilterOutlier, v7.4.5714.5459(7.4/embedded)
AucMatchUndercut, v7.4.5714.5585(7.4/embedded)
AucScanData, v7.4.5714 (TasmanianThylacine)
AucStatHistogram, v7.4.5714 (TasmanianThylacine)
AucStatiLevel, v7.4.5714 (TasmanianThylacine)
AucStatPurchased, v7.4.5714 (TasmanianThylacine)
AucStatSales, v7.4.5714.5598(7.4/embedded)
AucStatSimple, v7.4.5714 (TasmanianThylacine)
AucStatStdDev, v7.4.5714 (TasmanianThylacine)
AucStatWOWEcon, v7.4.5714.5613(7.4/embedded)
AucUtilAHWindowControl, v7.4.5714.5575(7.4/embedded)
AucUtilAppraiser, v7.4.5714.5704(7.4/embedded)
AucUtilAskPrice, v7.4.5714.5599(7.4/embedded)
AucUtilAutoMagic, v7.4.5714.5686(7.4/embedded)
AucUtilCompactUI, v7.4.5714.5694(7.4/embedded)
AucUtilEasyBuyout, v7.4.5714.5576(7.4/embedded)
AucUtilFixAH, v7.4.5714 (TasmanianThylacine)
AucUtilItemSuggest, v7.4.5714.5651(7.4/embedded)
AucUtilPriceLevel, v7.4.5714.5633(7.4/embedded)
AucUtilScanButton, v7.4.5714.5631(7.4/embedded)
AucUtilScanFinish, v7.4.5714.5588(7.4/embedded)
AucUtilScanProgress, v7.4.5714.4979(7.4/embedded)
AucUtilScanStart, v7.4.5714.5347(7.4/embedded)
AucUtilSearchUI, v7.4.5714.5710(7.4/embedded)
AucUtilSimpleAuction, v7.4.5714.5705(7.4/embedded)
AucUtilVendMarkup, v7.4.5714.4828(7.4/embedded)
Babylonian, v5.1.DEV.332(/embedded)
BagSync, v10.9
BarrelsOEasy, v1.0.0
Bartender4, v4.7.7
BattlePetCount, v1.8.13.1
BeanCounter, v7.4.5714 (TasmanianThylacine)
BlizzMove, v
BrokerCurrencyflow, vv1.6
BrokerDurabilityInfo, vv1.26
BrokerGarrison, v2.0.8.1-5-gc327e38
BrokerWorldQuests, v7.2.37
ChocolateBar, vv3.0.4
Configator, v5.1.DEV.406(/embedded)
DebugLib, v5.1.DEV.337(/embedded)
FishingBuddy, v1.7.13a
FlyoutButtonCustom, v2.79
Gatherer, v7.0.5
GearHelper, v1.6.5.2
Grail, v085
GrailAchievements, v011
GrailQuestsenUS, v22996
GrailReputations, v007
HandyNotes, vv1.4.13
HandyNotesLegionRaresTreasures, v1.01b
HandyNotesSuramarTelemancy, v1
HealBot, v7.1.0.2
HealBotTips, v7.1.0.2
IceHUD, v1.10.15
Informant, v7.4.5714 (TasmanianThylacine)
KibsItemLevel, v7.1.0.2
LibExtraTip, v5.12.DEV.423(/embedded)
Mapster, v1.7.5
Masque, v7.2.0
MasqueDiabloIII, v1.6
MinimapButtonFrame, v3.1.10e
NomiCakes, v7.2.0.1
NPCScan, v7.2.0.1
OmniCC, v7.1.1
Ovale, v7.2.0.0
ParagonReputation, v1.09
PerlArcaneBar, v
PerlCombatDisplay, v
PerlConfig, v
PerlFocus, v
PerlParty, v
PerlPartyPet, v
PerlPartyTarget, v
PerlPlayer, v
PerlPlayerPet, v
PerlTarget, v
PerlTargetTarget, v
PetBattleTeams, v3.3.7
PetJournalEnhanced, v2.9.26
PetsAndMounts, v1.8.7
PetsAndMountsLoader, v
PetTracker, v7.1.4
PetTrackerBreeds, v7.1.4
PetTrackerConfig, v7.1.4
PetTrackerJournal, v7.1.4
PetTrackerSwitcher, v7.1.4
PetTrackerUpgrades, v7.1.4
PhanxBuffs, v7.1.0.0
Postal, vv3.5.8
Rarity, v1.0
RarityOptions, v
Recount, vv7.2.0b release
SexyMap, vv7.2.0
SlideBar, v7.4.5714 (TasmanianThylacine)
SpellFlash, v7.03.22345
SpellFlashCore, v2.1.5
Stubby, v7.4.5714 (TasmanianThylacine)
TipHelper, v5.12.DEV.405(/embedded)
TomTom, vv70200-1.0.0
Vendomatic, v
Wholly, v062
WorldFlightMap, v7.2.0.1
Wyr3dIcyVeinsStats, v
XLoot, v7.2-1
XLootFrame, v7.2-1
BlizRuntimeLib_enUS v7.2.0.70200 
(ck=e3e)
